cmake/add_enclave.cmake

```diff
@@ -121,15 +122,23 @@ endfunction()
 
 function(add_enclave_sgx)
   set(options CXX)
-  set(oneValueArgs TARGET CONFIG KEY SIGNING_ENGINE ENGINE_LOAD_PATH ENGINE_KEY_ID)
+  set(oneValueArgs TARGET CONFIG KEY SIGNING_ENGINE ENGINE_LOAD_PATH ENGINE_KEY_ID ENABLE_LVI_MITIGATION)
   set(multiValueArgs SOURCES)
   cmake_parse_arguments(ENCLAVE
     "${options}"
     "${oneValueArgs}"
     "${multiValueArgs}"
     ${ARGN})
 
-  add_enclave_executable(${ENCLAVE_TARGET} ${ENCLAVE_SOURCES})
+  # Add an enclave with LVI mitigation when the `ENABLE_LVI_MITIGATION` argument or the `ENABLE_LVI_MITIGATION_TESTS`
+  # global variable is explicitly set.
+  if (LVI_MITIGATION MATCHES ControlFlow AND
+      ENCLAVE_ENABLE_LVI_MITIGATION OR ENABLE_LVI_MITIGATION_TESTS)
```

Comment:
does cmake allow parentheses to make the condition a little more clear? If so, add